tests/base/parser/bad/gold/cmodule-field-id-colon-cid-eq-hexlit-with.scilla.gold

```diff
@@ -1,8 +1,7 @@
 {
   "errors": [
     {
-      "error_message":
-        "Following a field definition, the parser expects another field definition, transition, procedure or end of file.\n",
+      "error_message": "Syntax error, state number 289",
```

Comment:
This was also discussed in private, I'm leaving a comment so that we don't forget about this issue.